firmware/smbios/smbios_bios_info.ksy

```diff
@@ -0,0 +1,42 @@
+meta:
+  id: smbios_bios_info
+  endian: le
+  imports:
+    - strings
+seq:
+  - id: length
+    type: u1
+  - id: handle_index
+    type: u2
+  - id: vendor_index
+    type: u1
+  - id: bios_version
+    type: u1
+  - id: bios_starting_address_segment
+    type: u2
+  - id: bios_release_date_index
+    type: u1
+  - id: bios_rom_size
+    type: u1
+  - id: bios_characteristics
+    type: u8
+  - id: bios_characteristics_ext_bytes
+    size: length - 0x12
```

Comment:
Again, if this will be size-bound substream, you could have just used `size-eos: true` on this attribute instead of introducing manual calculations.